andela/bp-esa-backend
# #162475201 Endpoint to fetch automation data

#### What does this PR do?
Implement an endpoint to retrieve automation results

#### Description of Task to be completed?
**When** I send a GET request to the endpoint
**Then** I should get a collection that contains data about the different automation that has happened.

#### How should this be manually tested?
1. Set the `TIMER_INTERVAL` to `20s` in your .env file
2. On node terminal, run `new Date()` and copy the date string
2. Make a request to mock-allocations endpoint to create new placement, using the generated date-string as your `created_at` field.
3. Wait for the automation to be carried out
4. On postman, make a `GET` request to: `localhost:8000/api/v1/automations`
5. Confirm that the created automation data were returned with `200` status code

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162475201](https://www.pivotaltracker.com/story/show/162475201)

#### Postman Documentation
LOADING...
